assets/src/containers/CourseList.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { withStyles } from '@material-ui/core/styles'
+import Grid from '@material-ui/core/Grid'
+import SelectCard from '../components/SelectCard'
+import Error from './Error'
+import { Link } from 'react-router-dom'
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1
+  },
+  container: {
+    display: 'flex',
+    justifyContent: 'center'
+  },
+  wrapper: {
+    maxWidth: 1023,
+    margin: theme.spacing.unit * 2 + 'px auto',
+    flexDirection: 'row',
+    justifyContent: 'flex-start'
+  }
+})
+
+function CourseList (props) {
+  const { classes } = props
+  // currently JSON.parse is required because Django is returning string instead of object
+  const userCourseInfo = JSON.parse(myla_globals.user_courses_info)
```

Comment:
I think this can't be returned as an object. since the context processor returning a dictionary so it is technically a string. Context processor won't return a response object as a typical API call does. So we need to do a JSON.parse to convert it into an object.